### PR TITLE
📚 New docs theme & Add comments

### LINKED ISSUE #72: move examples to jupyterbook + comments`
I was thinking that it'd be nice to add some commenting features to our example notebooks. 

For example, if a reader had questions about using xgcm with pop output. 

Jupyter book has some features for this: https://jupyterbook.org/interactive/comments.html so maybe it'd be good to try.

Alternatively, we could use Github Discussions but then the discussion is decoupled from the notebook...

### PR DESCRIPTION
**Pull request by andersy005**
- Closes #72 by adding comments and annotation functionality via sphinx-comments
- `sphinx_rtd_theme` --> `sphinx-book-theme`
- uses myst-nb/myst-parser 
- Docs can be written in markdown, ipynb and (rst if need be).

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -123,7 +148,30 @@
...
+    use_edit_page_button=True,
+    use_repository_button=True,
+    use_issues_button=True,
+    home_page_in_toc=False,
+    extra_navbar='',
+    navbar_footer_text='',
+    extra_footer="""Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>""",
+)
+
 html_logo = 'images/nsf.png'
```
Comment (andersy005) [inline docs/source/conf.py:175]: @mnlevy1981, does POP have a logo we could use here? If so, could you point me to it when you get a moment?

**Comment by andersy005** [comment]
@dcherian, here's the new layout with comments enabled:

Screen Shot 2021-03-15 at 8 21 13 AM

File: docs/source/conf.py (hunk 1 above)
Comment (mnlevy1981) [reply docs/source/conf.py:175]: I don't think there is one, but I've asked one of the LANL developers to be sure. I'll let you know if I hear anything back.

Comment (andersy005) [reply docs/source/conf.py:175]: Thanks!

**Comment by dcherian** [comment]
LGTM. I think it's worth trying the comment feature. Not sure whether anyone will use it, butit's free!

**Review by dcherian — APPROVED** [review]

**Pull request merged**